.changeset/tender-jeans-clean.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Can you please include an example of how this will be used? So examples of how to use the card and then how to add the extra properties as well. This 2nd part might need to be in the actual documentation though as I suspect it could be very long. This is part of the release notes so having information on how to use it is very helpful.

Reply:
yes sure this might take a while because the adding of catalog users (in my findings) was a little light in the documentation! In fact I reverse engineered the [`Roadie` Okta package](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/backend/catalog-backend-module-okta) to do but I'll see what I can do!

Reply:
would `https://backstage.io/docs/features/software-catalog/external-integrations` be the place to add this or a new section?

Reply:
done in external-integrations, happy to change as needed.

Reply:
Awesome, thanks!